SPECS/nodejs/CVE-2023-35945.patch

```diff
@@ -1,87 +0,0 @@
-From 5dc13a41fea5804e5b28478fa6cd07f31604965c Mon Sep 17 00:00:00 2001
```

Comment:
Patch no longer applies as nhttp2 has been updated to 1.57.0 in nodejs 20.10.0 https://github.com/nodejs/node/blob/v20.10.0/deps/nghttp2/lib/includes/nghttp2/nghttp2ver.h#L32, and https://nvd.nist.gov/vuln/detail/CVE-2023-35945 only affects up to 1.55.1 (https://github.com/nghttp2/nghttp2/pull/1930/commits/ce385d3f55a4b76da976b3bdf71fe2deddf315ba)